src/KlaviyoAPI.php

```diff
@@ -379,13 +380,21 @@ protected function setSinceParameter( $since, $uuid )
      * @param array $params
      * @return array
      */
-    protected function filterParams( array $params )
+    protected function filterParams( $params )
     {
-        return array_filter(
-            $params,
-            function ( $key ){ return !is_null( $key ); },
-            ARRAY_FILTER_USE_BOTH
-        );
+        if (version_compare(phpversion(), '5.6', '>')) {
+            return array_filter(
+                $params,
+                function ( $key ){ return !is_null( $key ); },
+                ARRAY_FILTER_USE_BOTH
+            );
+        } else {
+            return array_filter(
+                $params,
+                function ( $key ){ return !is_null( $key ); }
+            );
+        }
```

Comment:
as I read through this I think we can just scrap the array_filter_both constant. by default array_filter only passes the value (of a key/value pair) into the callback function which is all we care about in this method. we're just stripping out params that have a value set to null. in that case we can remove the conditional